components/textfield/stories/textfield.stories.js

```diff
@@ -122,24 +141,133 @@ export default {
 	},
 };
 
+/**
+ * Text fields should always have a label. In rare cases where context is sufficient and an accessibility expert has reviewed the design, the label could be undefined. These text fields without a visible label should still include an aria-label in HTML (depending on the context, “aria-label” or “aria-labelledby”).
+*/
+
 export const Default = TextFieldGroup.bind({});
-Default.args = {
-	labelText: "Username",
-};
+Default.args = {};
```

Comment:
What would you think about adding a "Standard" story (`Standard.storyName = "Default"`) to show all of the default focus, hover, invalid states? Kind of like you did for quiet? Those are still missing from the docs page, but it looks like they're in the controls. 👍

Then we could remove the invalid story completely, and just lump it in with all of the other states.